Dockerfile

```diff
@@ -2,7 +2,10 @@ FROM adoptopenjdk/openjdk13:debianslim
 
 # Install dependencies
 RUN apt-get update \
- && apt-get install -y --no-install-recommends git ca-certificates
+ && apt-get install -y --no-install-recommends git ca-certificates \
+ && rm -rf /var/lib/apt/lists/* \
+ && rm -rf /usr/share/doc && rm -rf /usr/share/man \
+ && apt-get clean
```

Comment:
There's no real reason for this. Since it's a multistage build, the build image wont be used for anything anyway, so cleaning up doesn't result in a smaller running image.

Reply:
Yes, it does not have any impact on final image, but I have not read in docker official documentation that you should not do this in build image, and it will help the project later on, if they want to add a linter for it.